bundles/org.openhab.binding.yamahamusiccast/src/main/java/org/openhab/binding/yamahamusiccast/internal/YamahaMusiccastBridgeHandler.java

```diff
@@ -0,0 +1,110 @@
+/**
+ * Copyright (c) 2010-2020 Contributors to the openHAB project
+ *
+ * See the NOTICE file(s) distributed with this work for additional
+ * information.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ */
+package org.openhab.binding.yamahamusiccast.internal;
+
+import java.util.concurrent.ScheduledExecutorService;
+import java.util.concurrent.ScheduledFuture;
+import java.util.concurrent.TimeUnit;
+
+import org.eclipse.jdt.annotation.NonNullByDefault;
+import org.eclipse.jdt.annotation.Nullable;
+import org.openhab.binding.yamahamusiccast.internal.model.UdpMessage;
+import org.openhab.core.common.ThreadPoolManager;
+import org.openhab.core.thing.*;
+import org.openhab.core.thing.binding.BaseBridgeHandler;
+import org.openhab.core.types.Command;
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import com.google.gson.Gson;
+
+/**
+ * The {@link YamahaMusiccastBridgeHandler} is responsible for dispatching UDP events to linked Things.
+ *
+ * @author Lennert Coopman - Initial contribution
+ */
+@NonNullByDefault
+public class YamahaMusiccastBridgeHandler extends BaseBridgeHandler {
+
+    private final Logger logger = LoggerFactory.getLogger(YamahaMusiccastBridgeHandler.class);
+
+    private final ScheduledExecutorService udpScheduler = ThreadPoolManager
+            .getScheduledPool("YamahaMusiccastListener" + "-" + thing.getUID().getId());
+    private @Nullable ScheduledFuture<?> listenerJob;
+    private final UdpListener udpListener;
+
+    public YamahaMusiccastBridgeHandler(Bridge bridge) {
+        super(bridge);
+        udpListener = new UdpListener(this);
+    }
+
+    @Override
+    public void handleCommand(ChannelUID channelUID, Command command) {
+    }
+
+    @Override
+    public void initialize() {
+        updateStatus(ThingStatus.ONLINE);
+        startUDPListenerJob();
+    }
+
+    @Override
+    public void dispose() {
+        stopUDPListenerJob();
+        super.dispose();
+    }
+
+    private void startUDPListenerJob() {
+        logger.info("YXC - Bridge Listener to start in 5 seconds");
```

Comment:
Logging to info should be used sparsely e.g. a newly started component or a user file that has been loaded. This could be debug.

Reply:
Set dot Debug as requested